### DISCUSSION TITLE
[Script request] OneDev

### DISCUSSION THREAD

**Discussion by ither** [community-scripts/ProxmoxVE#47] [Request script] [answered]
### Application Name

OneDev

### Website

https://onedev.io

### Description

Git server with CI/CD, kanban, and packages. Similar to Forgejo, the community version have everything to start managing large code projects. I think it shold not be dificult to implement as a script.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by quantumryuu** [comment, marked as answer]
@MickLesk will pick it up with the embedded DB.

**Reply by MickLesk** [reply]
Can you check the Project Dashboard, If you can write?

If yes, drag it to in Progress and assign to you, then create an issue of this and lets Go 😅
